internal/api/ws/codeocean_writer.go

```diff
@@ -114,6 +114,10 @@ func (cw *codeOceanOutputWriter) Close(info *runner.ExitInfo) {
 		message := "the allocation stopped as expected"
 		log.WithContext(cw.ctx).WithError(info.Err).Trace(message)
 		cw.send(&dto.WebSocketMessage{Type: dto.WebSocketOutputError, Data: message})
+	case errors.Is(info.Err, runner.ErrDestroyedAndReplaced):
+		errorMessage := "Runner recovery stopped request execution"
```

Comment:
request or requested? The one is related to the HTTP term (and technically correct), but I find the other one easier to read. 

```suggestion
		errorMessage := "Runner recovery stopped requested execution"
```